alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+                type: array
+                items:
+                  type: string
+                example:
+                  - keyword1
+                  - keyword2
+              properties:
+                description: Key-value-pair metadata associated with an alarm instance. Alarm instances can be tagged with properties so that they can be more easily found with queries. Additionally, alarm instance properties are used to expand message templates in the notification strategies associated with a given alarm instance.
+                type: object
+                example:
+                  key1: value1
+          required: true
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            title: ID of the created or modified instance
+            properties:
+              instanceId:
+                description: The ID of the created or modified alarm instance.
+                type: string
+                example: 5c40ec55e0d6441168b4c543
+        401:
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/Error'
+        409:
+          description: For requests which would result in an update to an existing alarm instance, this is returned when the request does not represent a valid transition for the existing alarm instance. This can occur when attempting to clear an alarm instance which is already clear, or when attempting to set an alarm instance which is already set at the given severity level.
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          $ref: '#/responses/Error'
+  /v1/instances/{id}:
+    parameters:
+      - in: path
+        name: id
+        description: Instance ID
+        type: string
+        required: true
+    get:
+      tags: [alarmInstances]
+      summary: Get an alarm instance
+      description: Gets an alarm instance by its *instanceId*
+      responses:
+        200:
+          $ref: '#/definitions/AlarmInstance'
+    delete:
+      tags: [alarmInstances]
+      summary: Delete an alarm instance
+      description: Deletes an alarm instance by its *instanceId*
+      responses:
+        204:
+          description: No Content
+        401:
+          $ref: '#/responses/Unauthorized'        
+        default:
+          $ref: '#/responses/Error'
+  /v1/instances/{id}/notes:
+    parameters:
+      - in: path
+        name: id
+        description: Instance ID
+        type: string
+        required: true
+    post:
+      tags: [alarmInstances]
+      summary: Add notes to an alarm instance
+      description: Adds one or more notes to an alarm instance.
+      operationId: AddNotesToInstance
+      parameters:
+        - in: body
+          name: Request body
+          required: true
+          schema:
+            type: object
+            required:
+              - notes
+            properties:
+              notes:
+                type: array
```

Comment:
It's pretty self-explanatory, but should this property have a description anyway?

Reply:
Added one